### PR TITLE
Create thumbnail version and show on user profile.

### PR DESCRIPTION
**Pull request by Ru-T**
## What Changed?
- Limited image size to thumbnail using RMagick
## Why?
- To avoid showing insanely massive photos

### CODE REVIEW

File: app/views/users/show.html.erb [hunk 1]
```diff
@@ -12,7 +12,7 @@
   <strong>Bio:</strong>
   <%= @user.bio %>
 </p>
-<div id="avatar"><%= image_tag(@user.avatar_url) if @user.avatar? %></div>
+<div id="avatar"><%= image_tag(@user.avatar_url(:thumb)) if @user.avatar? %></div>
```
Comment (dacur) [inline app/views/users/show.html.erb:15]: is `avatar_url` an `rmagick` helper method?  i couldn't find it defined anywhere.  i'm guessing it is  :)

**Comment by dacur** [comment]
just the one question!

otherwise, :shipit:

File: app/views/users/show.html.erb (hunk 1 above)
Comment (Ru-T) [inline app/views/users/show.html.erb:15]: @dacur avatar is my field name, and url is just how to render the image. that is actually carrierwave. i installed rmagick in order to generate a thumbnail. 

do you know of another way?

Comment (dacur) [inline app/views/users/show.html.erb:15]: @Ru-T ahh, i searched the rmagick docs instead of carrierwave!  sounds good!  :ship:

**Pull request merged**